src/reaktoro_pse/tutorials/basic_reaktoro_block_interaction.ipynb

```diff
@@ -678,33 +652,38 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": null,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "DOFs: 20\n",
+      "DOFs: 0\n",
       "Number of Reaktoro outputs 20\n",
-      "Actual DOFs: 0\n"
+      "Actual DOFs: -20\n"
+     ]
+    },
+    {
+     "ename": "AssertionError",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[1;31mAssertionError\u001b[0m                            Traceback (most recent call last)",
+      "Cell \u001b[1;32mIn[17], line 8\u001b[0m\n\u001b[0;32m      3\u001b[0m \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mNumber of Reaktoro outputs\u001b[39m\u001b[38;5;124m\"\u001b[39m, outputs_main_block)\n\u001b[0;32m      4\u001b[0m \u001b[38;5;28mprint\u001b[39m(\n\u001b[0;32m      5\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mActual DOFs:\u001b[39m\u001b[38;5;124m\"\u001b[39m,\n\u001b[0;32m      6\u001b[0m     degrees_of_freedom(m) \u001b[38;5;241m-\u001b[39m (outputs_main_block),\n\u001b[0;32m      7\u001b[0m )\n\u001b[1;32m----> 8\u001b[0m \u001b[38;5;28;01massert\u001b[39;00m degrees_of_freedom(m) \u001b[38;5;241m-\u001b[39m (outputs_main_block) \u001b[38;5;241m==\u001b[39m \u001b[38;5;241m0\u001b[39m\n",
+      "\u001b[1;31mAssertionError\u001b[0m: "
      ]
     }
    ],
    "source": [
     "print(\"DOFs:\", degrees_of_freedom(m))\n",
-    "outputs_main_block = len(m.fs.sea_water.eq_reaktoro_properties.reaktoro_model.outputs)\n",
-    "print(\"Number of Reaktoro outputs\", outputs_main_block)\n",
-    "print(\n",
-    "    \"Actual DOFs:\",\n",
-    "    degrees_of_freedom(m) - (outputs_main_block),\n",
-    ")\n",
-    "assert degrees_of_freedom(m) - (outputs_main_block) == 0"
+    "assert degrees_of_freedom(m)  == 0"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
+   "execution_count": null,
```

Comment:
This value being changed to `null` in this cell and subsequent ones seems to confirm the hypothesis that an error occurred in one of the previous cells.